sdk/core/azure-core/inc/azure/core.hpp

```diff
@@ -9,14 +9,14 @@
 #pragma once
 
 // azure/core
-#include "azure/core/azure_assert.hpp"
 #include "azure/core/base64.hpp"
 #include "azure/core/case_insensitive_containers.hpp"
 #include "azure/core/context.hpp"
 #include "azure/core/datetime.hpp"
 #include "azure/core/dll_import_export.hpp"
 #include "azure/core/etag.hpp"
 #include "azure/core/exception.hpp"
+#include "azure/core/internal/assert.hpp"
```

Comment:
Don't include internal headers in the simplified header that the end customer uses.
```suggestion
```

Reply:
Thank you good catch!